app/models/tasks/copy_media_task.rb

```diff
@@ -32,59 +32,25 @@ def task_status_changed(fixer_task, new_status)
   def task_options
     {
       job_type: 'audio',
-      destination: destination_url(media_resource),
-    }.merge(episode_options || story_options).with_indifferent_access
+      source: source_url(media_resource),
+      destination: destination_url(media_resource)
```

Comment:
this class is just a whole lot simpler when it assumes it can always use the `original_url`, nice to remove some code!